login.js

```diff
@@ -0,0 +1,5 @@
+const letsGo = document.getElementById('letsGo');
```

Comment:
`letsGo` has global scope.
 1. wrap file in function or
 2. just go `document.getElementById('letsGo').addEventListener('click', () => {`